Hearthstone Deck Tracker/Controls/DeckPicker/DeckPickerItemLayouts/DeckPickerItemLayout1.xaml

```diff
@@ -81,7 +81,7 @@
                 <DockPanel Margin="0,-2,0,0" Height="19">
                     <Viewbox DockPanel.Dock="Left" MinWidth="70" StretchDirection="DownOnly" Stretch="Fill"
                              HorizontalAlignment="Left">
-                        <TextBlock Text="{Binding StatsString, RelativeSource={RelativeSource AncestorType=deckPicker:DeckPickerItem}}" HorizontalAlignment="Left"
```

Comment:
:(

Reply:
Somebody recently had made these changes.. where they took out the RelativeSource=foo
I just merged it as is..  take it up with the other guy.  There were about 2-4 of these.

-  RelativeSource={RelativeSource AncestorType=deckPicker:DeckPickerItem}}"

Also there were a couple changes where  Margin="5" was removed on select .xaml files.
I only merged them!  Again.. not my doing!

Please advise if we need to revert back on those GUI changes?

 PS.

   I should have been up to date with merging everybody elses contributions.  I did not make changes to the GUIs.   In fact, other than the ResourceGenerator revamp and the refactoring of namespaces.. to match folder heirarchy..  everythign else should have been just optomizations with code analyzers and new format conventions as SAME!

The goal was to not change or break anything I didn't already have vetted.

Reply:
The RelativeSources were added, not deleted. Files which you are trying to push here are not updated. - In short version, you are deleting somebody work by pushing "old" files without latest changes. :P
Commit -  eb8876c4226265161f8a6da69ca6c62be5ec0f29